lib/libzutil/os/macos/zutil_import_os.c

```diff
@@ -135,15 +135,39 @@ zpool_open_func(void *arg)
 	dupname = zutil_strdup(hdl, rn->rn_name);
 	bname = basename(dupname);
 	error = ((strcmp(bname, "hpet") == 0) || is_watchdog_dev(bname));
+        if ((strncmp(bname, "core", 4) == 0) ||
+            (strncmp(bname, "fd", 2) == 0) ||
+            (strncmp(bname, "fuse", 4) == 0) ||
+            (strncmp(bname, "hpet", 4) == 0) ||
+            (strncmp(bname, "lp", 2) == 0) ||
+            (strncmp(bname, "parport", 7) == 0) ||
+            (strncmp(bname, "ppp", 3) == 0) ||
+            (strncmp(bname, "random", 6) == 0) ||
+            (strncmp(bname, "rtc", 3) == 0) ||
+            (strncmp(bname, "tty", 3) == 0) ||
```

Comment:
If you disable devices starting by "tty", I guess you can do the same with "cu".